.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow validates commit messages in a pull request to ensure they follow a specific pattern.
+# The pattern is defined as follows:
+# - The commit message must contain the string "[cherry-pick/XXX]" where XXX is a valid label.
```

Comment:
I'm a bit confused on this requirement, what does this provide? Is this something `git cherry-pick -x` does?

Reply:
No we would need to rebase and add [cherry-pick] cherry-pick -x simply adds the history comment at the base of the commit message

Reply:
Oh oh, I see. I was missing this was intended to be the subject of the commit message. I thought it was trying to enforce this in the body, which I didn't see why that added value over having the line about what commit it was cherry-picked from. I think Cherry-Pick in the commit subject makes sense (though only works if this is a rebase). What are you envisioning the XXX would be? Is `[Cherry-Pick]` not enough?

Reply:
My example right now is `[CHERRY-PICK/EDK2]` to signify where it was cherry-picked from however I'm willing to remove it. I wasn't sure if the source part was very useful

Reply:
We only have one upstream, so I don't think it is of much value. Maybe for tracking between Mu branches, but I would be of the opinion to just have `[CHERRY-PICK]` in the commit subject. The real goal for this, as I see it, is for the integrator of the next edk2 release to be able to easily drop the cherry-picks and reference them for the reverse operation (cherry-picking a potentially conflicting MU_CHANGE with what edk2 has).

Reply:
I agree with dropping the cherry-pick source for simplicity.